weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallDetails.tsx

```diff
@@ -132,7 +132,8 @@ export const CallDetails: FC<{
         }}>
         <Box
           sx={{
-            flex: '0 0 auto',
+            flex: '1 1 100px',
```

Comment:
This (and other similar changes) make it so that each section of the Call overview takes up equal space and does not scroll